### DISCUSSION TITLE
cosnole.log wird weder in der vs-code console, noch im browser ausgeloggt

### DISCUSSION THREAD

**Discussion by TobiasSteinhagen** [neuefische/questions#673] [Web: Self-directed learning] [answered]
Moin,

ich arbeite aktuell an einem eigenen Projekt und wollte nun das erste JS hinzufügen. Um zu prüfen,  ob ich auf dem richtigen Weg mit meinem Array bin, wollte ich mir dieses einmal ausloggen.

```js
const player = [
  {
    player: "Anna",
    score: "04",
  },
  {
    player: "Berna",
    score: "05",
  },
  {
    player: "Bonnie",
    score: "02",
  },
  {
    player: "David K.",
    score: "02",
  },
  {
    player: "David v.H.",
    score: "01",
  },
  {
    player: "Elli",
    score: "07",
  },
  {
    player: "Henrique",
    score: "02",
  },
  {
    player: "Jaoh",
    score: "02",
  },
  {
    player: "Leo",
    score: "02",
  },
  {
    player: "Marcus",
    score: "02",
  },
  {
    player: "Miriam",
    score: "01",
  },
  {
    player: "Pia",
    score: "03",
  },
  {
    player: "Ping",
    score: "02",
  },
  {
    player: "Shwan",
    score: "02",
  },
  {
    player: "Stephan",
    score: "03",
  },
  {
    player: "Tien",
    score: "02",
  },
  {
    player: "Tobi",
    score: "05",
  },
];

console.log(player);
```

Leider wird mit das weder in der Console in VS Code, noch in der Console im Browser ausgeloggt.

![Bildschirm­foto 2023-08-17 um 15 49 54](https://github.com/neuefische/questions/assets/140393240/35107d80-fddf-47b9-8e5d-528f0cce0d35)

Ich habe bereits gegoogelt, bin aber auf keine Lösung gestoßen. Mein Projekt findet ihr [hier](https://github.com/TobiasSteinhagen/deskgame). 

Hat jemand eine Idee?

Viele Grüße
Tobi

**Comment by mbosselmann** [comment, marked as answer]
Hallo Tobias,
du hast vergessen das `script` tag im `head` von deiner `index.html` einzubinden. Wenn du das noch einfügst (`<script src="./index.js" defer></script>`), dann ist die Verbindung zur JavaScript-Datei hergestellt. :) Du findest hierzu auch noch was im [Handout von der JS Basics Session](https://github.com/neuefische/bo-web-23-4/blob/main/sessions/js-basics/js-basics.md).

Ich hoffe, dass hilft dir weiter! 💪

**Reply by TobiasSteinhagen** [reply]
Danke dir, manchmal kann es so einfach sein....🤦‍♂️
